mrjob/emr.py

```diff
@@ -712,17 +717,10 @@ def _check_output_not_exists(self):
     def _add_bootstrap_files_for_upload(self, persistent=False):
         """Add files needed by the bootstrap script to self._upload_mgr.
 
-        Tar up mrjob if bootstrap_mrjob is True.
-
         Create the master bootstrap script if necessary.
 
         persistent -- set by make_persistent_cluster()
         """
-        # lazily create mrjob.zip
```

Comment:
Why are we removing this?

Reply:
Because mrjob is no longer part of bootstrapping; it's now part of `py_files` like on the Hadoop runner.